assets/src/edit-story/components/canvas/layout.js

```diff
@@ -390,6 +410,15 @@ const PageArea = forwardRef(function PageArea(
       data-scroll-container
       {...rest}
     >
+      {enableRightClickMenus && (
+        <RightClickMenuContainer position={menuPosition}>
+          <AnimatedContextMenu
+            isOpen={isMenuOpen}
+            onDismiss={onCloseMenu}
+            items={rightClickMenuItems}
+          />
+        </RightClickMenuContainer>
+      )}
```

Comment:
This feels a little weird to me since these these components feel like they're supposed to be mostly presentational. Thoughts?

Reply:
You could put all of this in its own component like
```
 {enableRightClickMenus && <RightClickMenu /> }
```

Reply:
I'm fine with @BrittanyIRL 's suggestion or leaving it as is. Have no preference here really